raw_data_QA/check_raw_data.py

```diff
@@ -0,0 +1,78 @@
+import pandas as pd
+import matplotlib.pyplot as plt
+import numpy as np
+from sklearn.metrics import mean_squared_error
+from datetime import datetime
+import calendar
+import argparse
+
+def aggregate_df(df, args):
+  df[args.new_cases_name] = pd.to_numeric(df[args.new_cases_name])
+  df[args.new_deaths_name] = pd.to_numeric(df[args.new_deaths_name])
+  aggregation_functions = {args.new_cases_name:'sum', args.new_deaths_name:'sum', args.updated_date_name:'first'}
+  df_new = df.groupby(df[args.date_name]).aggregate(aggregation_functions)
+  df_new['new_cases'] = df_new[args.new_cases_name].diff().fillna(0)
+  df_new['new_deaths'] = df_new[args.new_deaths_name].diff().fillna(0)
+  return df_new
+
+def get_state(df, state, args):
+  df_new = df.loc[df[args.state_name] == state]
+  df_new[args.updated_date_name] = df[[args.date_name]]
+  return df_new
+
+def get_rmse(df1, df2, var):
+  df1_length = len(df1.index)
+  truncated_df2 = df2.head(df1_length)
+  rmse = np.sqrt(mean_squared_error(df1[[var]], truncated_df2[[var]]))
+  return round(rmse,2)
+
+def make_plot(var, df1, df2, df3, df1_name, df2_name, df3_name, args, state):
+  rmse2 = get_rmse(df1, df2, var)
+  rmse3 = get_rmse(df2, df2, var)
+  plt.title(state)
+  plt.xlabel(var)
+  plt.ylabel(args.updated_date_name)
+  plt.plot(df1[args.updated_date_name], df1[var], color = 'blue', label = df1_name + ' NYT, RMSE: ' + str(rmse2), markersize = 8, marker = '.', alpha = 0.5)
+  plt.plot(df2[args.updated_date_name], df2[var],  color = 'orange', label = df2_name + ' NYT, RMSE: ' + str(rmse3), markersize = 8, marker = '.', alpha = 0.5)
+  #plt.plot(df3[args.updated_date_name], df3[var], alpha = 0.5, color = 'green', label = df3_name + ' NYT: RMSE: ' + str(rmse3), markersize = 10, marker = '.')
+  plt.xticks(rotation=30)
+  plt.legend(loc = 'upper left')
+  plt.grid(True)
+  plt.savefig(state + '_raw_' +  var + '_compare.pdf', bbox_inches='tight')
+  plt.close('all')
+
+def get_current_day():
+  current_month = calendar.month_name[datetime.now().month]
+  current_day = datetime.now().day
+  return current_month + ' ' + str(current_day)
+
+
+if __name__ == '__main__':
+  parser = argparse.ArgumentParser(description = 'arg parser for process.py')
+  #Output Dir------
+  parser.add_argument('-output_dir', '--output_dir', type = str, dest = 'output_dir', default = 'output', help = 'output_dir')
+  parser.add_argument('-state', '--state', type = list, dest = 'state', default = ['Idaho', 'Vermont', 'Texas'], help = 'name of state to process')
```

Comment:
does this work with type=list? I think this gets a bit funky if you want to override this: https://stackoverflow.com/questions/15753701/how-can-i-pass-a-list-as-a-command-line-argument-with-argparse

Reply:
Good point, I changed this in the last commit